wiki/Gameplay/Input_device/Wiimote/en.md

```diff
@@ -0,0 +1,9 @@
+# Wiimote
+
+The **Wiimote** is the primary game controller for the Nintendo Wii console.  
+The Wiimote can be used in osu! to aim by using the sensor bar and you can rebind any button to click in the osu! options.
+Be sure to first enable support for it in the osu! options. 
+
+![Wiimote support option in osu!](img/1.jpg "Wiimote support option in osu!")
```

Comment:
This image doesn't work; link leads to 404 error

Reply:
works for me, it did give 404 when looking from the pr but you can try looking from en.md instead of pr.